ArduPlane/Attitude.cpp

```diff
@@ -455,7 +455,13 @@ void Plane::calc_nav_yaw_coordinated(float speed_scaler)
         commanded_rudder = yawController.get_servo_out(speed_scaler, disable_integrator);
 
         // add in rudder mixing from roll
-        commanded_rudder += SRV_Channels::get_output_scaled(SRV_Channel::k_aileron) * g.kff_rudder_mix;
+        float rudder_mix;
+        if (flight_stage == AP_Vehicle::FixedWing::FLIGHT_TAKEOFF && g.takeoff_rudder_mix_height > (-1.0f*relative_altitude) && g.takeoff_rudder_mix_height > 0) {
```

Comment:
@bugobliterator it's more for the clarity of what feature the if statement is looking to enable/control. Everything else in the line is logic related, the > 0 check is the enable, and gives the reader a good hint of why the rest of the code is being run.